### DISCUSSION TITLE
It doesn't work.

### DISCUSSION THREAD

**Discussion by LaykaKiro** [SteamRE/DepotDownloader#290] [Q&A]
At this point this post will just sit here as it's no longer a question. No amount of help worked because even when I did everything properly this trash didn't work. As far as I'm concerned, it really is some sort of false hope at this point.

**Comment by GoTeamScotch** [comment]
Not sure if there was a temporary bug or something, but I downloaded DepotDownloader for the first time yesterday and it worked perfectly.

What command are you using?

Example:
`DepotDownloader.exe -app 123100 -depot 123101 -manifest 103544760119053156 -username %YourUsername% -password %YourSteamPassword% -dir "%YourInstallPath%"`

**Reply by LaykaKiro** [reply]
I know I never replied to this. But I used the appropriate command, filled in the details correctly and it didn't work.

**Comment by milroneth** [comment]
This issue can be safely closed, as per #288 OP doesn't understand what command line tool means. They explained "doesn't work" as double clicking the exe disappears too fast.

**Reply by LaykaKiro** [reply]
Given that I followed all given instruction damn near flawlessly and this didn't work. I do genuinely believe one of the releases is broken or it's non functional. And a lot of people are sucking some fat copium about it.

**Reply by milroneth** [reply]
An issue tracker is not a good place for arguments, but because, even though we actually use it, you insist that we think it works, but you know better and we are delusional:

It works. I use it. It downloads my stuff. I have downloads sitting right here. The current latest version works fine, and the latest version when you opened the issue also worked fine then.

I will not pretend it's easy to use or it has the best documentation, it is very hard and it was the only tool in the last years that took me so long to figure out and I'm in this field in a professional context for 15 years and have been using command line tools since I was 8.

As a regular user, you are not obliged to know it all or be a command line master by any means, and if you believe that the documentation is lacking, you can report that (but if you do and you want actual help and not just to throw your overly confident ignorance to us, please include specifics, including EXACTLY what you tried to run that didn't work so that folks can find what's missing).

But regardless, this issue and your other one claiming "It doesn't work" need to be closed because they are simply false and contribute nothing in making this software better. It doesn't matter what you think and if you believed you followed everything flawlessly, if it didn't work for you, then you didn't. Unless you provide specifics and prove that something really doesn't work.

**Reply by GoTeamScotch** [reply]
I agree. Issue can likely be closed due to user error.

Even still, we could probably help guide them if only we had more information.

[LaykaKiro](https://github.com/LaykaKiro) Explain step by step what you did. What game were you trying to download files for? What's an example of the command you tried running? What was the response from the app? Perhaps share screenshots. Saying "I tried it and it's broke" doesn't give us nearly enough info to help. Like taking your car in for repairs and only telling them "it's broken". Please be more descriptive.
